griptape/drivers/vector/pgai_knowledge_base_vector_store_driver.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING, NoReturn, Optional
+
+from attrs import Factory, define, field
+
+from griptape.artifacts import ImageArtifact, ListArtifact, TextArtifact
+from griptape.drivers.embedding.dummy import DummyEmbeddingDriver
+from griptape.drivers.vector import BaseVectorStoreDriver
+from griptape.utils import import_optional_dependency
+from griptape.utils.decorators import lazy_property
+
+if TYPE_CHECKING:
+    import sqlalchemy
+
+    from griptape.drivers.embedding import BaseEmbeddingDriver
+
+logger = logging.getLogger(__name__)
+
+
+@define
+class PGAIKnowledgeBaseVectorStoreDriver(BaseVectorStoreDriver):
```

Comment:
```suggestion
class PgAiKnowledgeBaseVectorStoreDriver(BaseVectorStoreDriver):
```

For consistency with `PgVectorVectorStoreDriver` and `OpenAiChatPromptDriver`